plugins/by-name/dap/dapHelpers.nix

```diff
@@ -98,7 +98,11 @@ rec {
 
   mkAdapterOption =
     name: type:
-    mkNullOrOption (with types; attrsOf (either str type)) ''
+    let
+      # TODO: Added 2025-12-11 (26.05)
+      strToRawLua = lib.nixvim.deprecation.transitionType types.str lib.nixvim.mkRaw types.rawLua;
```

Comment:
I thought we had a helper for this kind of behavior but didn't see it, at first glance.